TemperatureConvertor.java

```diff
@@ -4,7 +4,9 @@
 public class TemperatureConvertor {
 
     public static void main(String[] args) {
-
+/*
+        Am adaugat textul acesta pentru review
+ */
```

Comment:
odata mergeuit eu mia vad doar acest text:) trebuie sa nu dai merge pana nu revin eu. odata mergeuit e parte din main si nu se mai vad schimbarile ca si schimbari.

Reply:
Am adugat metoda the inchidere a scanner -ului